arch/arc/core/mpu/arc_mpu.c

```diff
@@ -59,7 +59,6 @@ static inline u8_t _get_num_regions(void)
 	return (u8_t)num;
 }
 
-
```

Comment:
not related to the changes in this PR, but not sure you can/should define a new local variable inside a case block  (u8_t bits in _get_region_attr_by_type below)